src/components/select/CustomSelectDropdown.tsx

```diff
@@ -86,47 +89,69 @@ const CustomSelectDropdown: FC<Props> = ({
 
   // track selected index from key board action and scroll into view if needed
   useEffect(() => {
-    if (selectedIndex !== -1 && optionsRef.current[selectedIndex]) {
+    if (optionsRef.current[selectedIndex]) {
```

Comment:
I think we can simplify now to `optionsRef.current[selectedIndex]?.scrollIntoView...`